### PR TITLE
Implement Nancy Parser

### PR DESCRIPTION
**Pull request by grendel513**
JSON file parser for the Nancy scanner: https://github.com/sonatype-nexus-community/nancy

[sc-4916]

### CODE REVIEW

File: dojo/tools/nancy/parser.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        return findings
+
+    def get_items(self, vulnerable, test):
+        findings = []
+        for vuln in vulnerable:
+            finding = None
+            severity = 'Info'
+            # the tool does not define severity, however it
+            # provides CVSSv3 vector which will calculate
+            # severity dunamically on save()
```
Comment (cneill) [inline dojo/tools/nancy/parser.py:41]:
```suggestion
            # severity dynamically on save()
```

File: dojo/tools/nancy/parser.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        component_name=comp_name,
+                        component_version=comp_version,
+                        false_p=False,
+                        duplicate=False,
+                        out_of_scope=False,
+                        static_finding=True,
+                        dynamic_finding=False,
+                        vuln_id_from_tool=associated_vuln["Id"],
+                        cve=associated_vuln['Cve'],
+                        references=", ".join(references),
```
Comment (cneill) [inline dojo/tools/nancy/parser.py:67]:
```suggestion
                        references="\n".join(references),
```

**Review by cneill — COMMENTED** [review]
Just a couple minor suggestions here

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by grendel513** [comment]
> Just a couple minor suggestions here

thank you @cneill - committed your suggestions

**Review by cneill — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**